10/RequestReply.md

```diff
@@ -76,6 +76,15 @@ SHOULD include:
 
 #### Exception
 
+Certain conditions may prevent a Request to be handled properly.
+
+* If a Request cannot be handled properly, the Client SHALL be notified by receiving of a DevFailed or MultiDevFailed exception (see [9/DataTypes.md](/9/DataTypes.md#Exceptions)).
```

Comment:
Is there a definition of a MultiDevFailed exception somewhere?

Reply:
This is missing in RFC-9.

Reply:
I have added #115 to make sure it will be added

Reply:
There is #116 fixing #115 :-), @bourtemb or @hardion, please review and merge if you find it is ok.